frontend/templates/product-list/sections/FilterableProductsSection/RefinementMultiSelect.tsx

```diff
@@ -92,4 +92,5 @@ const MultiSelectItem = React.memo(function MultiSelectItem({
          </Text>
       </HStack>
    );
+   return null;
```

Comment:
do we need these return nulls?

Reply:
Thanks for pointing it out! You are right we do not need these anymore.
Resolved 5e36b200fbc107858ffc45e7501bab9f0d45b083